### PR TITLE
[Tables] Add sas token live tests

### PR DESCRIPTION
**Pull request by YalinLi0312**
For issue: https://github.com/Azure/azure-sdk-for-python/issues/28918

### CODE REVIEW

File: sdk/tables/azure-data-tables/azure/data/tables/_models.py [hunk 1]
```diff
@@ -541,7 +541,8 @@ class ResourceTypes(object):
     def __init__(self, **kwargs) -> None:
```
Comment (kristapratico) [inline sdk/tables/azure-data-tables/azure/data/tables/_models.py:541]: `container` is missing in the docstring

Comment (kristapratico) [reply sdk/tables/azure-data-tables/azure/data/tables/_models.py:541]: Also maybe silly question, but how does a `container` resource type work with tables? Is there a container concept in tables, or is the expectation that you could create the sas in this library and use it across storage libraries?

Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/azure/data/tables/_models.py:541]: - Actually the term `container` here is a litter confused for me, from the test result it's the accessibility required for creating Tables.
- The sas token generated here can also be used in other libraries that want to access the same storage account.

Comment (YalinLi0312) [reply sdk/tables/azure-data-tables/azure/data/tables/_models.py:543]: > `container` is missing in the docstring

Added, thanks for catching!

**Review by kristapratico — APPROVED** [review]

**Pull request merged**